source/blog/tags.html

```diff
@@ -8,6 +8,17 @@
 
 <div>
 {% for tag,posts in data.posts_tags %}
-<a href="{{ site.url }}/blog/tags/{{ tag|url_encode(true) }}">{{ tag }}</a>
+
+<!-- conditions to hide author tag names on the tags page --> 
+  {% if tag == "nikki-stevens" %} 
+    <a class="hide" href="{{ site.url }}/blog/tags/{{ tag|url_encode(true) }}">{{ tag }}</a> 
+  {% elseif tag == "mike-klass" %}
+     <a class="hide" href="{{ site.url }}/blog/tags/{{ tag|url_encode(true) }}">{{ tag }}</a> 
+  {% elseif tag == "jason-savino" %}
+     <a class="hide" href="{{ site.url }}/blog/tags/{{ tag|url_encode(true) }}">{{ tag }}</a>
+ 
+  {% else %} 
+  <a href="{{ site.url }}/blog/tags/{{ tag|url_encode(true) }}">{{ tag }}</a>
```

Comment:
oh let's also add something to the readme here about how when a new author is added, they will have to be added to this exception list.